x-pack/plugin/wildcard/src/main/java/org/elasticsearch/xpack/wildcard/mapper/regex/Or.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.wildcard.mapper.regex;
```

Comment:
This all might want to be in `wildcard.mapper.automaton` or something like that. We use it for all kinds of stuff.